R/count_bindings.R

```diff
@@ -652,3 +652,44 @@ keep_levels <- function(e, ...) {
   e
 }
 
+#' Set a numeric cutoff
+#'
+#' In certain tables, it may be necessary to only include rows that meet numeric
+#' conditions. Rows that are less than a certain cutoff can be suppressed from
+#' the output. This function allows you to pass a cutoff, a cutoff stat(n,
+#' distinct_n, pct, or distinct_pct) to supress values that are lesser than the
+#' cutoff.
+#'
+#' @param e A \code{count_layer} object
+#' @param numeric_cutoff A numeric value where only values greater than or equal
+#'   to will be displayed.
+#' @param stat The statistic to use when filtering out rows. Either 'n',
+#'   'distinct_n', or 'pct' are allowable
+#' @param column If only a particular column should be used to cutoff values, it
+#'   can be supplied here as a character value.
+#'
+#' @return The modified Tplyr layer object
+#' @export
+#'
+#' @examples
+#' mtcars %>%
+#' tplyr_table(gear) %>%
+#'   add_layer(
+#'     group_count(cyl) %>%
+#'       set_numeric_where(10, "n") %>%
+#'       add_total_row() %>%
+#'       set_order_count_method("bycount")
+#'   )
+set_numeric_where <- function(e, numeric_cutoff, stat, column = NULL) {
```

Comment:
Based on my previous comment, I could see this argument as 

```suggestion
set_numeric_where <- function(e, filter_condition) {
```